olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```
@@ -832,31 +832,31 @@ spec:
+                        value: "registry.ci.openshift.org/openshift/knative-serving-activator:knative-v1.9"
                       - name: "IMAGE_autoscaler"
-                        value: "registry.ci.openshift.org/openshift/knative-serving-autoscaler:knative-v1.8"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-autoscaler:knative-v1.9"
                       - name: "IMAGE_autoscaler-hpa"
-                        value: "registry.ci.openshift.org/openshift/knative-serving-autoscaler-hpa:knative-v1.8"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-autoscaler-hpa:knative-v1.9"
                       - name: "IMAGE_controller__controller"
-                        value: "registry.ci.openshift.org/openshift/knative-serving-controller:knative-v1.8"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-controller:knative-v1.9"
                       - name: "IMAGE_webhook__webhook"
-                        value: "registry.ci.openshift.org/openshift/knative-serving-webhook:knative-v1.8"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-webhook:knative-v1.9"
                       - name: "IMAGE_domain-mapping"
-                        value: "registry.ci.openshift.org/openshift/knative-serving-domain-mapping:knative-v1.8"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-domain-mapping:knative-v1.9"
                       - name: "IMAGE_domainmapping-webhook"
-                        value: "registry.ci.openshift.org/openshift/knative-serving-domain-mapping-webhook:knative-v1.8"
-                      - name: "IMAGE_storage-version-migration-serving-serving-1.8__migrate"
-                        value: "registry.ci.openshift.org/openshift/knative-serving-storage-version-migration:knative-v1.8"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-domain-mapping-webhook:knative-v1.9"
+                      - name: "IMAGE_storage-version-migration-serving-serving-1.9__migrate"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-storage-version-migration:knative-v1.9"
                       - name: "IMAGE_kourier-gateway"
                         value: "quay.io/maistra-dev/proxyv2-ubi8:2.3-latest"
                       - name: "IMAGE_net-kourier-controller__controller"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:kourier"
+                        value: "registry.ci.openshift.org/openshift/net-kourier-kourier:knative-v1.9"
                       - name: "IMAGE_net-istio-controller__controller"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:net-istio-controller"
+                        value: "registry.ci.openshift.org/openshift/net-istio-controller:knative-v1.9"
```

```suggestion
                        value: "registry.ci.openshift.org/openshift/net-istio-controller:knative-1.9"
```
I think no `v` for this image.